overlord/snapstate/handlers_test.go

```diff
@@ -392,6 +393,32 @@ func (s *handlersSuite) TestGetHiddenDirOptionsNoState(c *C) {
 	c.Check(opts, DeepEquals, &snapstate.DirMigrationOptions{UseHidden: true})
 }
 
+func (s *handlersSuite) TestDoEnforceValidationSetsTask(c *C) {
```

Comment:
this is fine but I think it was covered already also by some more complete preexisting test